docs/conf.py

```diff
@@ -108,7 +108,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+html_theme = 'classic'
```

Comment:
I would be fine with the warning as long as it allows certain build
environments to inject their preferred theme (ie readthedocs)

On Sat, 5 Dec 2015 3:16 AM Ashley Whetter <notifications@github.com> wrote:

> In docs/conf.py
> <https://github.com/sqlboy/fileseq/pull/35#discussion_r46685374>:
>
> > @@ -108,7 +108,7 @@
> >
> >  # The theme to use for HTML and HTML Help pages.  See the documentation for
> >  # a list of builtin themes.
> > -html_theme = 'default'
> > +html_theme = 'classic'
>
> I've just noticed that this change has forced the template on readthedocs
> to use the classic template instead of the readthedocs template. How do you
> want to handle this?
> If this is switched back to default it creates a warning when the docs are
> built. If we switch it to the readthedocs template then it requires a
> dependency on the sphinx_rtd_theme module.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sqlboy/fileseq/pull/35/files#r46685374>.
>